src/main/resources/com/google/api/codegen/ruby/smoke_test.snip

```diff
@@ -23,7 +23,15 @@ describe "{@smokeTest.name}" do
       project_id = ENV["SMOKE_TEST_PROJECT"].freeze
     @end
 
-    {@sampleCodeCore(smokeTest.apiMethod)}
+    process_response = ->(response) {
```

Comment:
That works only for basic unary calls, and it was how I was doing it initially. How do you propose we handle [these other cases](https://github.com/googleapis/toolkit/pull/793#discussion_r105246194) though?